controllers/secret/secret_controller.go

```diff
@@ -180,7 +184,13 @@ func (r *ReconcileSecret) Reconcile(request reconcile.Request) (reconcile.Result
 			return reconcile.Result{}, errors.Wrap(err, "error creating signer from private key")
 		}
 		nodes := &core.NodeList{}
-		err = r.client.List(context.TODO(), nodes, client.HasLabels{nodeconfig.WindowsOSLabel})
```

Comment:
Can't you do the equivalent of  `tc.client.K8s.CoreV1().Nodes().List(context.TODO(), metav1.ListOptions{LabelSelector: nodeconfig.WindowsOSLabel})` that we have in our e2e tests? i.e use the label selector.

Reply:
I ended up taking @VaishnaviHire suggestion and using the core.LabelOSStable label which makes this simpler

Reply:
While I am fine with the change, it has again been made silently. Please call this out in the commit message. Also add a follow up story to replace `nodeconfig.WindowsOSLabel` with `core.LabelOSStable`